keyboards/tada68/keymaps/rupa/config.h

```diff
@@ -0,0 +1,11 @@
+#include "../../config.h"
```

Comment:
Other `config.h` files are automatically included 
```suggestion
#pragma once
```